packages/sources/linkpool/src/config.ts

```diff
@@ -0,0 +1,12 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { Config } from '@chainlink/types'
+
+export const NAME = 'LinkPool'
+
+export const DEFAULT_BASE_URL = 'https://api.ice.linkpool.io/v1/futures/'
```

Comment:
I'm guessing `futures` is a part of a specific endpoint and not the base URL across all endpoints.